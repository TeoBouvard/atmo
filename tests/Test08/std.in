2
O3
2017-01-01
2017-01-10
5
