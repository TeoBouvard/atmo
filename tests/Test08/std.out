Importation des capteurs ... 3 capteurs importÃ©s
Importation des mesures ... 12 mesures importÃ©es

Menu :
 1 : ConnaÃ®tre la qualitÃ© de lâ€™air sur une zone donnÃ©e
 2 : Identifier les zones ayant une qualitÃ© de lâ€™air similaire
 3 : ConnaÃ®tre la qualitÃ© de l'air en un point donnÃ©
 4 : Identifier les capteurs dÃ©fectueux
 5 : Quitter


Choisir un polluant (O3 | NO2 | SO2 | PM10) : Date de dÃ©but au format YYYY-MM-DD : Date de fin au format YYYY-MM-DD : 
Analyse en cours ... 

[1m[37mNombre de capteurs comparÃ©s : 3
Nombre de mesures sur cette pÃ©riode de temps : 1[0m

           |   0  |   1  |   2  |
Sensor nÂ°0 | [1m[32m100%[0m | [1m[31m  0%[0m | [1m[32m 75%[0m | 

Sensor nÂ°1 | [1m[31m  0%[0m | [1m[32m100%[0m | [1m[31m 24%[0m | 

Sensor nÂ°2 | [1m[32m 75%[0m | [1m[31m 24%[0m | [1m[32m100%[0m | 


Analyse terminÃ©e.

Menu :
 1 : ConnaÃ®tre la qualitÃ© de lâ€™air sur une zone donnÃ©e
 2 : Identifier les zones ayant une qualitÃ© de lâ€™air similaire
 3 : ConnaÃ®tre la qualitÃ© de l'air en un point donnÃ©
 4 : Identifier les capteurs dÃ©fectueux
 5 : Quitter


